packages/terra-avatar/src/Avatar.scss

```diff
@@ -0,0 +1,54 @@
+@import '~terra-mixins';
+
+:local {
+  .avatar {
+    align-items: center;
+    background-color: var(--terra-avatar-background-color, rgb(222, 223, 224));
+    border: var(--terra-avatar-border);
+    border-radius: 50%;
+    box-shadow: var(--terra-avatar-box-shadow);
+    color: var(--terra-avatar-color, rgb(155, 159, 161));
+    display: inline-flex;
+    height: var(--terra-avatar-size, 3.429rem);
+    justify-content: center;
+    line-height: 0;
+    overflow: hidden;
+    width: var(--terra-avatar-size, 3.429rem);
+  }
+  
+  .avatar-icon {
+    height: 100%;
+    width: 100%;
+    
+    > svg {
```

Comment:
**NOTE:** if we end up changing the tag like I mentioned above on this to a span, then these styles will no longer be applied, as we won't be using the `svg` element. I think it might be better to have a target class at the icon anyway, that the `facility` and `user` css classes are nested within, to ensure they only have the potential to target the element your inline-svg is being set as the background on, but i'm not super picky about this

Reply:
Updated in this [commit](https://github.com/cerner/terra-core/commit/18d816503ca37ccd585667ee835ecd2b04377d96).